src/nuget/Microsoft.DotNet.BuildTools.TestSuite.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.TestSuite</id>
```

Comment:
Sorry, I meant for this comment to be in Microsoft.xunit.extensibility.execution.netcore, not testSuite.